graphql_test.go

```diff
@@ -3217,14 +3226,29 @@ func TestInputArguments_failSchemaParsing(t *testing.T) {
 			},
 			Want: want{Error: "field \"Input\": *struct { Thing string } does not define field \"name\" (hint: missing `args struct { ... }` wrapper for field arguments, or missing field on input struct)\n\tused by (*graphql_test.inputArgumentsObjectMismatch3).Hello"},
 		},
+		"Struct field name inclusion": {
+			Args: args{
+				Resolver: &helloOutput{},
```

Comment:
According to your schema you need to have a field or a method named `Hello`, which returns an instance of `helloOutput`. Instead you are using the `helloOutput` instance as a resolver. This doesn't make sense to me.

Reply:
Woops, messed up that while writing the test. Fixing it.